docker/puppetserver/spec/puppetserver_spec.rb

```diff
@@ -29,14 +31,11 @@ def puppetserver_health_check(container)
 
   it 'should start puppetserver successfully' do
     status = puppetserver_health_check(@container)
-    while status == 'starting'
+    while (status == 'starting' || status == "'starting'")
       sleep(1)
       status = puppetserver_health_check(@container)
     end
-    unless status == 'healthy'
-      puts %x(docker logs #{@container})
-    end
-    expect(status).to eq('healthy')
+    expect(status).to match(/\'?healthy\'?/)
```

Comment:
Do we get anything useful out of `docker logs` when things fail?

Reply:
mm, yeah, that would probably be helpful for actually figuring out what went wrong if something failed, will add that back